plugins/processors/snmp_lookup/README.md

```diff
@@ -0,0 +1,130 @@
+# SNMP Lookup Processor Plugin
+
+The `snmp_lookup` plugin looks up extra tags using SNMP and caches them.
+
+Telegraf minimum version: Telegraf 1.29.0
```

Comment:
```suggestion
Telegraf minimum version: Telegraf 1.30.0
```

Reply:
Oh right, just a leftover because this PR got delayed so much.